lib/imagemagick.js

```diff
@@ -15,7 +15,8 @@
 var sprintf = require('util').format,
   log = require('sslog'),
   lru = require("lru-cache"),
-  child = require('child_process');
+  child = require('child_process')
+  playButtonPath = __dirname + '/../public/play-button.svg';
```

Comment:
I thought about pulling it out to configs, but I felt that it made more sense inline. The play button refers to a file within mkimage-server. Pulling the file's path into the configs would require the configs to have knowledge of the file system structure of the machine/container it's running on. Also to me it felt more similar to a `require` than a config. And since there's only one play button I can't think of anything that would need to be configured. But I'm happy to continue discussing.